src/test/java/cucumber/steps/StepDef.java

```diff
@@ -100,10 +108,96 @@ public void loadPage_for_Welsh_Expenses(String publicBureau, String env) throws
 
 
 	@Then("^I log in as \"([^\"]*)\"$")
-	public void loginWith_usernameOnly(String username) throws Throwable {
-		LGN.login(username, System.getenv(username + "_PASSWORD"));
+	public void loginWithUsername(String username) throws Throwable {
+		loginWithAD(username, "");
+	}
+
+
+	@Then("^I log in as \"([^\"]*)\" selecting court \"([^\"]*)\"$")
+	public void loginWithUsernameAndCourt(String username, String specifiedCourt) throws Throwable {
+		loginWithAD(username, specifiedCourt);
+	}
+
+	public void loginWithAD(String username, String specifiedCourt) throws Throwable {
+		String defaultCourt = null;
+		switch (username) {
+			case "MODTESTBUREAU":
+			case "SYSTEMADMIN":
+			case "AUTO":
+			case "TeamPickListUser":
+			case "AutomationStaffTWO":
+			case "SYSTEM":
+			case "SJOUSER1":
+			case "SJOUSER":
+			case "CPASS":
+			case "ARAMIS1":
+			case "CMANAGER":
+			case "CMANAGER2":
+			case "SHREWSBURY":
+			case "NEWUSER":
+				LGN.loginADTestRoute(username + "@email.gov.uk");
+				break;
+			case "MODTESTCOURT":
+				defaultCourt = "415";
+				LGN.loginADTestRoute(username + "@email.gov.uk");
+				break;
+			case "MODCOURT":
```

Comment:
Don't take my word for this but I think you can probably do:

```
case "MODTESTCOURT": case "MODCOURT":
	defaultCourt = "415";
	LGN.loginADTestRoute(username + "@email.gov.uk");
	break;
```

Reply:
wasn't sure if different users were needed to have different default courts, so left this as a bit of an example for that, although probably should have added some comments to explain that